### DISCUSSION TITLE
how to make avatar upload api for current user?

### DISCUSSION THREAD

**Discussion by Martian95** [directus/directus#19302] [Feature Requests]
### Summary

how to make avatar upload api for current user?

### Basic Example

POST users/me

### Motivation

no

### Detailed Design

no

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

no

### Alternatives

no

### Adoption Strategy

no

### Unresolved Questions

no

**Comment by rijkvanzanten** [comment]
Closing this as it's not a feature request. [Discord](https://directus.chat) is the better place for questions 👍🏻
